BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m

```diff
@@ -50,9 +50,16 @@ - (BOXAPIOperation *)createOperation
     NSDictionary *queryParameters = nil;
 
     if (self.requestAllFileFields) {
-        queryParameters = @{BOXAPIParameterKeyFields: [self fullFileFieldsParameterString]};
-    }
+        NSString *fieldString = [self fullFileFieldsParameterString];
+        
+        // Include the representations field in the request supporting the request for file representation
+        if ([self.httpHeaderValueXRepHints length] > 0) {
```

Comment:
This changed per our team discussion. We will use a bool to build the high definition video request. The comments are updated to support the change.